lib/event_store/store/postgres.ex

```diff
@@ -58,16 +76,16 @@ defmodule EventStore.Store.Postgres do
     |> Repo.transaction()
     |> case do
          {:error, _multi_key, _cs, _res} = err -> retry_error(err)
-         {:ok, %{} = res} -> Map.values(res)
+         {:ok, _} -> :ok
        end
   end
 
   defp append_changeset(cs, mult),
-    do: Multi.insert(mult, changeset_key(cs), cs, returning: true)
+    do: Multi.insert(mult, changeset_key(cs), cs)
 
   defp retry_error({:error, _, %{errors: [sequence: {:dupe_seq_agg, _}]}, _}),
     do: {:error, :retry_command}
-  defp retry_error(err), do: raise EventStore.StoreError.exception(err)
```

Comment:
I'm not sure how I feel about raising here. Is it possible to return `{:error, StoreError.exception(err)}` and raise it at the call site?